### PR TITLE
m4acut: Init at 0.1.2

### PR DESCRIPTION
**Pull request by chkno**
###### Motivation for this change
Make m4acut available.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
    0 → 39,646,096
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/m4acut/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    sha256 = "1hzf9f1fzmlpnxjaxhs2w22wzb28vd87ycaddnix1mmhvh3nvzkd";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = [ l-smash ];
+
+  meta = with lib; {
+    description = "Losslessly & gaplessly cut m4a (AAC in MP4) files.";
+    homepage = "https://github.com/nu774/m4acut";
+    license = with licenes; [ bsdOriginal zlib ];
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/m4acut/default.nix:19]:
```suggestion
    license = with licenses; [ bsdOriginal zlib ];
```

File: pkgs/applications/audio/m4acut/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{ autoreconfHook, fetchFromGitHub, l-smash, lib, stdenv }:
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/m4acut/default.nix:1]:
```suggestion
{ lib, stdenv, autoreconfHook, fetchFromGitHub, l-smash }:
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/audio/m4acut/default.nix (hunk 1 above)
Comment (chkno) [reply pkgs/applications/audio/m4acut/default.nix]: Done.  Thanks!

File: pkgs/applications/audio/m4acut/default.nix (hunk 2 above)
Comment (chkno) [reply pkgs/applications/audio/m4acut/default.nix]: Done.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 110660` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>m4acut</li>
</ul>
</details>

File: pkgs/applications/audio/m4acut/default.nix [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{ lib, stdenv, autoreconfHook, fetchFromGitHub, l-smash }:
+stdenv.mkDerivation rec {
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/m4acut/default.nix:2]:
```suggestion
{ lib, stdenv, autoreconfHook, fetchFromGitHub, l-smash }:

stdenv.mkDerivation rec {
```

**Pull request merged**